packages/paste-core/components/combobox/src/multiselect/GrowingInput.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```

Comment:
Question: should the growingInput have any unit tests? Just to make sure the width is right or something?

Reply:
Wasn't sure what I could test tbh. It's all CSS, so nothing to unit test. And visually it's invisible, so not much to VRT either.